### ISSUE TITLE
Upload multiple files to a nodejs server

### ISSUE THREAD

**Issue by vancuf** [KasemJaffer/receive_sharing_intent#189]
How do I format the links to upload  multiple files to a NodeJS server? So far I can only upload files one by one. 
the URLs format for multiple share is one long URL "/data/user/0/com.example.share_app/cache/20210615_213948.jpg,/data/user/0/com.example.share_app/cache/Screenshot_2018-05-11-20-38-25.png"

**Comment by GourabPaul007** [comment]
this is not this package specific question?

but you can run a for loop for every file path in _sharedFiles and add the links to a list.
may be this can help you [https://stackoverflow.com/a/63269887](url)

**Comment by vancuf** [comment]
> this is not this package specific question?
> 
> but you can run a for loop for every file path in _sharedFiles and add the links to a list. may be this can help you [https://stackoverflow.com/a/63269887](url)

The stackover link not working.

**Comment by GourabPaul007** [comment]
here you go [https://stackoverflow.com/questions/63263840/how-to-upload-multiple-images-files-in-flutter-using-dio/63269887#63269887](url)

**Comment by GourabPaul007** [comment]
https://stackoverflow.com/questions/63263840/how-to-upload-multiple-images-files-in-flutter-using-dio/63269887#63269887

**Comment by vancuf** [comment]
> this should work https://stackoverflow.com/questions/63263840/how-to-upload-multiple-images-files-in-flutter-using-dio/63269887#63269887

Yes this works with one url but the problem i am having is breaking up multiple urls "/data/user/0/com.example.share_app/cache/20210615_213948.jpg,/data/user/0/com.example.share_app/cache/Screenshot_2018-05-11-20-38-25.png" then uploading with dio

**Comment by GourabPaul007** [comment]
you can split the string with String's split(","); method. Or better - dont  merge the string in first place. make a list of string and when maping through the _sharedFiles, add each file path to the list.

**Comment by bhagat-techind** [comment]
You can use my plugin [flutter_sharing_intent](https://pub.dev/packages/flutter_sharing_intent)
